bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -115,11 +113,16 @@ public boolean apply(CheckAndMutateRowRequest checkAndMutateRowRequest) {
         }
       };
 
-  private static final Predicate<SampleRowKeysRequest> IS_RETRYABLE_SAMPLE_ROW_KEY =
-      Predicates.<SampleRowKeysRequest> alwaysTrue();
+  private static final boolean allCellsHaveTimestamps(Iterable<Mutation> mutations) {
+    for (Mutation mut : mutations) {
+      if (mut.getSetCell().getTimestampMicros() == -1) {
```

Comment:
I changed this.  I hope that this doesn't bite some of our users who may have not set timestamp.